Bot/MessageHandlers/Commands/strike.js

```diff
@@ -0,0 +1,84 @@
+const ROLES = {
+    LUCKY : "902322660262957056"
+    STRIKEONE : "896026304237932565",
+    STRIKETWO : "906695092230832139",
+    BANNED : "614522888372092928",
+}
+
+function getMemberFromMention(mention, channel) {
+	if (mention.startsWith("<@") && mention.endsWith(">")) {
+		mention = mention.slice(2, -1);
+
+		if (mention.startsWith("!")) {
+            mention = mention.slice(1);
+			const foundMember = channel.members.find(
+                (member) => member.user.id === mention
+            );
+            return foundMember;
+		}
+
+        return undefined;
+	}
+}
+
+function addRoleById(msg, target, roleId) {
+    let role = msg.guild.roles.cache.find(r => r.id === roleId);
+    target.roles.add(role)
+        .then(() => {
+            console.log(`Given ${role} to <@${target.user.id}>`);
+        })
+        .catch(console.error);
+}
+
+function removeRoleById(msg, target, roleId) {
+    let role = msg.guild.roles.cache.find(r => r.id === roleId);
+    target.roles.remove(role)
+        .then(() => {
+            console.log(`Given ${role} to <@${target.user.id}>`);
+        })
+        .catch(console.error);
+}
+
+module.exports = {
+	name: "strike",
+	help: "Strike",
+	command: async function (msg, args) {
+        let body = args.join(" ");
+        let targetMember = getMemberFromMention(body, msg.channel);
+        if (!targetMember) {
+            msg.channel.send(`Could not find member: ${target}`);
```

Comment:
I would just not send anything here and have the bot do nothing. Since @'ing in Discord would bring up the available users anyway, so the only way the `getMemberFromMention` function would fail is if they type it out manually. In which case they probably aren't using the command right